### DISCUSSION TITLE
webUI username/password issue

### DISCUSSION THREAD

**Discussion by Gr33nMach1ne** [telekom-security/tpotce#1620] [General]
Hello,

So I am currently experiencing an issue with gaining access to the webUI via port 64294. When attempting to login, I am unable to gain access. My webuser credentials work when accessing kibana via 64297 but despite combing over the documentation and attempts to try and update the login info, I cannot login despite creating new users via the nginx instructions.

**Comment by t3chn0m4g3** [comment]
This is port is reserved for incoming log data from a sensor...
![image](https://github.com/user-attachments/assets/28ea216d-53cd-402a-9bfc-8005fb655dff)

When deploying sensors according to the ReadMe users / passwords will be added as part of the enrollment.
